additional.conf

```diff
@@ -0,0 +1,10 @@
+# Optional custom command or file collection for configsnap
+#
```

Comment:
It may be slightly duplicating documentation but I would add a description here including valid types in here. Eg:

```
# Each config block must be in the following foemat:
# [filename]
# Type: [ file | command ]
# [ File | Command ]: [ source filename | full command to be run ]
# Fail_ok: [ true | false ]
# Sort: [ true | false ]